bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -146,7 +169,7 @@
 			</parameter>
 			<parameter name="macOnline" type="text" multiple="true">
 				<label>MAC Online</label>
-				<description>List of MACs for "online" status detection (format: 11:11:11:11:11:11). One MAC-Address per row.</description>
+				<description>List of MACs for "online" status detection (format: 11:11:11:11:11:11).</description>
```

Comment:
This was recently fixed, please revert.

Reply:
Why is that necessary?

Reply:
Because the documentation was unclear. Details here https://github.com/openhab/openhab-addons/pull/14291 and https://github.com/openhab/openhab-addons/issues/13798

Reply:
I don't understand why one would like to add a comma-separated list there, but ok.